code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -4001,15 +4008,7 @@
 	if(..())
 		return 1
 
-	if(volume >= 3)
-		T.wet(800)
-	var/hotspot = (locate(/obj/effect/fire) in T)
-	if(hotspot)
-		var/datum/gas_mixture/lowertemp = T.remove_air(T:air:total_moles())
-		lowertemp.temperature = max( min(lowertemp.temperature-2000,lowertemp.temperature / 2), 0)
-		lowertemp.react()
-		T.assume_air(lowertemp)
-		qdel(hotspot)
+	call(/datum/reagent/water/proc/douse)(T, volume)
```

Comment:
This makes me uncomfortable
why is that a `/datum/reagent/water` proc anyway?

Reply:
Didn't mean to leave it like that. Whoops.